release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1269,6 +1269,17 @@ Optionally, you can now override the default evaluation intervals for all polici
 
 For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-configuring-pgt-compliance-eval-timeouts_ztp-advanced-policy-config[Configuring policy compliance evaluation timeouts for PolicyGenTemplate CRs].
 
+id=["specifying-the-platform-type-for-managed-clusters"]
+==== Specifying the platform type for managed clusters
+
+The Assisted Installer currently supports the following OpenShift platforms:
+
+* `BareMetal`
+* `VSphere`
+* `None`
+
+Single node OpenShift does not support `VSphere`. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc#specifying-the-platform-type-for-managed-clusters_ztp-deploying-far-edge-clusters-at-scale[Specifying the platform type for managed clusters].
```

Comment:
vsphere platform. Neither BareMetal. but not sure we should talk in those terms here -  do we have none/baremetal as part of the ztp api?

Reply:
@imiller0 -- Does the GitOps ZTP plugin support different platforms? Or is this something we shouldn't document as a part of the network far edge documentation?

Reply:
The GitOps flow (using SiteConfig) does not directly expose the platformType. I don't think there is a need to reference the far-edge section here. 

Separately, the section being pointed to does not seem to fit in with the far-edge documentation either. We should review that for purpose/flow and consider removing it. I'll open a separate docs story for that.

Reply:
@danmacpherson, just confirming that your question has been answered and any suggested changes?

Reply:
Yep, we'll need to remove following:

"For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc#specifying-the-platform-type-for-managed-clusters_ztp-deploying-far-edge-clusters-at-scale[Specifying the platform type for managed clusters]." 

We'll also need to find a new section for the associated documentation, or possibly remove the content temporarily (as per @imiller0's doc story)

Reply:
Updated as suggested.